.github/workflows/README.md

```diff
@@ -0,0 +1,51 @@
+# GitHub Workflows
+
+This directory contains automated workflows for the HomeRacker repository.
+
+## Workflows
+
+### 1. Handle Release PRs (`release-please.yml`)
+Creates and updates release PRs based on conventional commits.
```

Comment:
maybe also hint that the release pr accumulates until it's getting merged.
and this happens through the automerge release pr